magma/spec/gnomon_spec.rb

```diff
@@ -439,6 +439,83 @@ def app
     end
   end
 
+  context 'bulk generate API' do
+
+    it 'throws exception when no grammar for project' do
+      auth_header(:admin)
+      json_post('/gnomon/labors/generate', names: [])
+      expect(last_response.status).to eq(422)
+      expect(json_body[:error]).to eq('No grammar found for project labors.')
+    end
+
+    context 'with grammar' do
+      before(:each) do
```

Comment:
is `:each` needed here? Isn't the default behavior for each spec? Unless you like the explicitness

Reply:
hmm, is that the default behavior? I didn't know that... but i like the explicitness anyway